retriever/lib/defaults.py

```diff
@@ -23,6 +26,13 @@
     os.path.join(HOME_DIR, 'raw_data/{dataset}'),
 ]
 DATA_WRITE_PATH = DATA_SEARCH_PATHS[-1]
+RETRIEVER_SCRIPTS = [
+"acton_lake.json",
+"amniote_life_hist.py",
+"bioclim.json",
+"iris.json",
+"predicts.py"
+]
```

Comment:
This is done to simply mark some of the scripts as retriever legacy scripts, which would be kept in the retriever repo itself and not shifted to the recipes repo.